src/lib/components/common/ContentEditable.js

```diff
@@ -42,11 +42,11 @@ export default class ContentEditable extends Component {
   };
 
   renderDiv = () => {
+    const { link,target, openWindowFeatures } = this.props.linkInfo;
```

Comment:
Can we revert to the old behaviour if linkInfo is not set? This way, we don't force these fields to be links if we don't want to.